src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Upgrade/ConsoleUpgrade.cs

```diff
@@ -61,6 +64,9 @@ public async Task RunAsync(CancellationToken token)
                 var steps = await _upgrader.InitializeAsync(context, token);
                 var step = await _upgrader.GetNextStepAsync(context, token);
 
+                using var outStream = File.Create("E:\\outa.txt");
```

Comment:
Just a test. Will be replaced with a path in the project root and format coming from ConsoleUpgradeOptions